### ISSUE TITLE
Error: 2024-12-14T21:34:39.824Z Fatal Error Report

### ISSUE THREAD

**Issue by JonathanTurnock** [flying-dice/dcs-dropzone-mod-manager#45]
This is an automated error report generated by the application.

- **Error:** Error
- **Message:** Error: ENOENT: no such file or directory, scandir 'C:\Users\jonat\AppData\Local\dcs-dropzone\mods'
- **Stack Trace:**
```text
Error: Error: ENOENT: no such file or directory, scandir 'C:\Users\jonat\AppData\Local\dcs-dropzone\mods'
    at process.<anonymous> (C:\Users\jonat\Projects\dcs-dropzone-mod-manager\out\main\index.js:2481:19)
    at process.emit (node:events:519:28)
    at emitUnhandledRejection (node:internal/process/promises:250:13)
    at warnWithErrorCodeUnhandledRejectionsMode (node:internal/process/promises:404:19)
    at processPromiseRejections (node:internal/process/promises:470:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
```
---
[Please describe what you were doing when the error occurred here.]

Please Upload the logs from the following location:
%APPDATA%\dcs-dropzone\logs\main.log

---
2024-12-14T21:34:39.287Z [InstanceLoader] [info] DiscoveryModule dependencies initialized
2024-12-14T21:34:39.287Z [InstanceLoader] [info] ScheduleModule dependencies initialized
2024-12-14T21:34:39.770Z [MongooseModuleFactory] [info] Dropzone db mongodb://127.0.0.1:57449/
2024-12-14T21:34:39.770Z [MongooseModuleFactory] [info] Dropzone db path C:\Users\jonat\AppData\Roaming\dcs-dropzone\__data
2024-12-14T21:34:39.780Z [InstanceLoader] [info] MongooseCoreModule dependencies initialized
2024-12-14T21:34:39.784Z [InstanceLoader] [info] MongooseModule dependencies initialized
2024-12-14T21:34:39.784Z [InstanceLoader] [info] AppModule dependencies initialized
2024-12-14T21:34:39.786Z [SubscriptionManager] [info] Checking for orphaned subscriptions and releases
2024-12-14T21:34:39.800Z [SubscriptionManager] [error] Method removeOrphanedSubscriptionsAndReleases failed after 14ms with error: ENOENT: no such file or directory, scandir 'C:\Users\jonat\AppData\Local\dcs-dropzone\mods'

---

**Comment by JonathanTurnock** [comment]
Duplicates #42

**Comment by JonathanTurnock** [comment]
Resolved in #46
